packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -202,9 +204,44 @@ suite('requests helpers', ({ expect, stub, spy }) => {
       expect(req.query).to.eq(overrideRequest.query);
       expect(req.refinements).to.eq(overrideRequest.refinements);
       expect(req.skip).to.eq(overrideRequest.skip);
-      expect(req.sort).to.eq(overrideRequest.sort);
       expect(req.extra).to.eq(overrideRequest.extra);
     });
+
+    it('should use the overrideRequest to generate the sanitized sort', () => {
+      const sanitizedSort = { baz: 'quux' };
+      const overrideRequest: any = {
+        sort: { foo: 'bar' },
+      };
+      stub(RequestAdapter, 'extractPastPurchaseSort').withArgs(overrideRequest.sort).returns(sanitizedSort);
+
+      const req: any = RequestHelpers.pastPurchaseProducts(state, overrideRequest);
+
+      expect(req.sort).to.eql(sanitizedSort);
+    });
+
+    it('should use the selected sort to generate the sanitized sort', () => {
+      const sanitizedSort = { baz: 'quux' };
+      const overrideRequest: any = {
+        a: 'b',
+      };
+      stub(RequestAdapter, 'extractPastPurchaseSort').withArgs(sort).returns(sanitizedSort);
+
+      const req: any = RequestHelpers.pastPurchaseProducts(state, overrideRequest);
+
+      expect(req.sort).to.eql(sanitizedSort);
+    });
+
+    SORT_FIELDS.forEach((field) => {
+      it(`should handle the "${field}" sort`, () => {
+        ppSortStub.returns({ field });
+
+        const req = RequestHelpers.pastPurchaseProducts(state);
+
+        expect(req.sort).to.eql({ type: 'ByIds', ids: [1, 2, 3] });
+
+        ppSortStub.returns(sort);
```

Comment:
ppSortStub seems to be only used inside of this "it" block, should we consider moving it down here instead of having it in the BeforeEach() if it isn't being used anywhere else?

Reply:
Ah I see, understood. Resolving feedback.